scripts/exregional_get_obs_mrms.sh

```diff
@@ -150,7 +154,7 @@ Did you forget to run \"module load hpss\"?\
     fi
 
   else
-    echo "mrms_file exists: \"$mrms_proc/${vyyyymmdd}/${field_base_name}${level}${vyyyy}${vmm}${vdd}-${vhh}0000.grib2\" No work to be done."
+    echo "mrms_file exists: \"$mrms_proc/${vyyyymmdd}/${field_base_name}${level}${vyyyy}${vmm}${vdd}-${}0000.grib2\" No work to be done."
```

Comment:
@michelleharrold While attempting to run the get_obs_mrms task on Hera, I'm seeing the following error message:

```
MRMS FILE:/scratch2/NAGAPE/epic/Michael.Lueken/UFS_SRW_data/develop/obs_data/mrms/proc/../proc/20200615/MergedReflectivityQCComposite_00.50_20200615-190000.grib2
/scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/scripts/exregional_get_obs_mrms.sh: line 157: mrms_file exists: "$mrms_proc/${vyyyymmdd}/${field_base_name}${level}${vyyyy}${vmm}${vdd}-${}0000.grib2" No work to be done.: bad substitution
```
Is there a reason that you removed vhh from the bracket on line 157?  This behavior has been seen for several different dates.

When I added vhh back in, the task successfully completes and the verification test runs through to completion.

Reply:
@MichaelLueken -- that must an error on my part (likely accidentally removed when I was searching for vhh due to the suggestion from @gsketefian to correct how we set vhh_noZero). I will fix that and push the mods. Thanks for catching that!

Reply:
Thanks for the modification, @michelleharrold!  Resolving conversation now.